ops/where/versions.go

```diff
@@ -0,0 +1,11 @@
+package where
+
+import "github.com/advancedclimatesystems/gonnx/ops"
+
+var whereVersions = ops.OperatorVersions{
+	9: ops.NewOperatorConstructor(newWhere, 9, whereTypeConstraints),
+}
+
+func GetVersions() ops.OperatorVersions {
```

Comment:
GetWhereVersions? Consistency with the rest?

Reply:
#228